airbyte-integrations/connectors/destination-postgres/src/main/java/io/airbyte/integrations/destination/postgres/PostgresDestination.java

```diff
@@ -65,12 +65,7 @@ public JsonNode toJdbcConfig(final JsonNode config) {
 
     String encodedDatabase = config.get(JdbcUtils.DATABASE_KEY).asText();
     if (encodedDatabase != null) {
-      try {
-        encodedDatabase = URLEncoder.encode(encodedDatabase, "UTF-8");
-      } catch (UnsupportedEncodingException e) {
-        // Should never happen
-        e.printStackTrace();
-      }
+      encodedDatabase = URLEncoder.encode(encodedDatabase, StandardCharsets.UTF_8);
```

Comment:
What is the rationale for removing this try/catch change? https://github.com/airbytehq/airbyte/pull/16046

This seems like it would be a breaking change since it touches `DestinationPostgres` and not `DestinationPostgresRds`, no?

Reply:
Not really. the reason why `UnsupportedEncodingException` is thrown in the first place is because you are providing an encoding as a String which might not exist. The ENUM solves that issue and the method does no longer throw the exception in the catch. And the if not null statement that you are referring to is still there.